docs/machine_and_deep_learning.md

```diff
@@ -10,6 +10,7 @@
 -   [Machine learning with scikit-learn 3](https://neurohackademy.org/course/machine-learning-with-scikit-learn-3/)  by Jake Vanderplas within Neurohackademy  [Video, ca. 140 mins]
 -   [Deep learning and machine learing tutorials](https://github.com/brainhack101/introML) from the Montreal Artificial Intelligence and Neuroscience conference, Montreal 2018 by many authors. [Jupyter notebook, 2 days]
 -   [TensorFlow 2.0 Complete Course - Python Neural Networks for Beginners](https://www.youtube.com/watch?time_continue=13&v=tPYj3fFJGjk) by Tim Ruscica (Tech with Tim) [Video, 7 hrs]
+* [An introduction to statistical learning](https://faculty.marshall.usc.edu/gareth-james/ISL/) , [python_examples](https://github.com/JWarmenhoven/ISLR-python)

```

Comment:
There are a few additions that would be great:
1. Authors must be cited
2. Name of the complete book should be mentioned 
3. Jupyter Notebooks would be much appropriate as per the repo